nexus-writer/src/nexus/run_parameters.rs

```diff
@@ -65,6 +65,20 @@ impl RunParameters {
         }
     }
 
+    #[tracing::instrument(skip_all, level = "trace", err(level = "warn"))]
+    pub(crate) fn set_emergency_stop(&mut self) -> anyhow::Result<()> {
+        if self.run_stop_parameters.is_some() {
+            anyhow::bail!("RunStop already set");
```

Comment:
https://github.com/STFC-ICD-Research-and-Design/supermusr-data-pipeline/blob/main/docs/style.md#crab-error-handling-rules

Reply:
In the "nexus writer refactor" issue I'm replacing all the non-main uses of `anyhow`, so am using that crate with the error handling to be consistent with the current usage.